airflow/providers/cncf/kubernetes/triggers/pod.py

```diff
@@ -263,6 +264,7 @@ async def _wait_for_container_completion(self) -> TriggerEvent:
                         "name": self.pod_name,
                     }
                 )
+            self.log.info("Sleeping for %s seconds.", self.poll_interval)
```

Comment:
I think it may make sense to split the logs between the running event we have in between. It may happen that once we return to operator with running, we might not return here if the pod completes by then while we're in the operator?